src/transformStateWithClones.js

```diff
@@ -7,7 +7,35 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const stateCopy = { ...state };
+  const stateArrCopy = [];
```

Comment:
let's use something more meanfull in variable names
this array is the history of states, also avoid data types in names

Reply:
✔️